dynamic-plugins/wrappers/backstage-plugin-azure-devops-backend-dynamic/dist-dynamic/package.json

```diff
@@ -44,10 +45,14 @@
   ],
   "bundleDependencies": true,
   "peerDependencies": {
-    "@backstage/backend-common": "0.19.8",
-    "@backstage/backend-plugin-api": "0.6.6",
-    "@backstage/backend-plugin-manager": "npm:@janus-idp/backend-plugin-manager@0.0.2-janus.5",
```

Comment:
+1 for getting rid of these `npm:` dependencies in favour of properly declared ones